workshops/templates/workshops/workshop_issues.html

```diff
@@ -36,7 +36,7 @@
         {% if event.mailto_ %}
           <a href="mailto:{{event.mailto_}}?subject={% filter urlencode %}Missing location for workshop {{event.get_ident}}{% endfilter %}&body={% filter urlencode %}Hi,
 
-Can you please send us the location where the {{ event.get_ident }} workshop took place?
+Can you please send us the location where the {{ event.get_ident }} workshop took place? We need a country, address, venue, and latitude/longitude.
```

Comment:
I don't think we want to add conditionals to the email. Now that I think about it, we might not want to change the wording at all - more text == less reading.

@maneesha can you please review this change?